internal/migrations/clusterlogforwarder/migrate_inputs.go

```diff
@@ -6,6 +6,21 @@ import (
 )
 
 func MigrateInputs(namespace, name string, spec loggingv1.ClusterLogForwarderSpec, logStore *loggingv1.LogStoreSpec, extras map[string]bool, logstoreSecretName, saTokenSecret string) (loggingv1.ClusterLogForwarderSpec, map[string]bool, []loggingv1.Condition) {
+	for i, input := range spec.Inputs {
+		if input.Receiver != nil {
+			if input.Receiver.HTTP != nil && len(input.Receiver.Type) == 0 {
+				input.Receiver.Type = loggingv1.ReceiverTypeHttp
+				spec.Inputs[i] = input
+				extras[constants.MigrateInputApplication] = true
```

Comment:
Do we need to set this here?  We did this in other places because otherwise it would fail a validation but I'm not certain its required

Reply:
removed